### ISSUE TITLE
Replication count works, but examples not shown

### ISSUE THREAD

**Issue by ebarry** [publiclab/plots2#1566]
### What happened just before the problem occurred

I visited https://publiclab.org/notes/liz/03-15-2017/host-a-balloon-mapping-workshop and noticed that replications weren't showing up anymore although the count was still 35. 

Did something change?

**Comment by jywarren** [comment]
No, this is just a bug that cropped up recently and I haven't had time to look into or fix it, apologies. I'll take a look, thanks for opening an issue as I'd forgotten to.

**Comment by jywarren** [comment]
So we need to look for rows like this in the tests for this feature:

```
<tr>
--
  | <td class="title"><a href="/notes/liz/03-15-2017/host-a-balloon-mapping-workshop">Host a balloon mapping workshop</a></td>
  | <td class="category">-</td>
  | <td class="status">-</td>
  | <td class="author"><a href="/profile/liz">@liz</a></td>
  | <td class="time">-</td>
  | <td class="difficulty">-</td>
  | <td class="replications">35 replications: <a href="/notes/liz/03-15-2017/host-a-balloon-mapping-workshop#replications">Try it &raquo;</a></td>
  | </tr>
```

The test is here: https://github.com/publiclab/plots2/blob/master/test/integration/node_insert_extras_test.rb#L32

**Comment by jywarren** [comment]
Trying to reproduce this in a test at #1567

**Comment by ebarry** [comment]
Here's another example of this known bug, where this activity https://publiclab.org/notes/cfastie/06-09-2017/the-mini-pearl-logger shows a count of one replication but has no link to the content (which is here: https://publiclab.org/anotes/Zengirl2/09-01-2017/mini-pearl-logger-replication)

cc @Fastie @zengirl2

**Comment by jywarren** [comment]
Sorry I've been too slammed to work on this. The pull request is partway there; hope to circle back and come up with some next steps soon.

**Comment by jywarren** [comment]
So for anyone looking to help with this, https://github.com/publiclab/plots2/pull/1567 now reproduces the error. We now need to figure out how to fix it, which would then solve the test failure shown in https://github.com/publiclab/plots2/pull/1567.

**Comment by Zengirl2** [comment]
Hey, just jumping in to correct the link for my post. Should be: https://publiclab.org/notes/Zengirl2/09-01-2017/mini-pearl-logger-replication.

**Comment by Zengirl2** [comment]
I don't know Rails, but just checking the tagging on the @Fastie original post and my post. 

- His _Mini Pearl Logger_ post- Response: 14152

- My _Mini Pearl Logger Replication_ post- Replication: 14518

My tag doesn't match the correct number I wrote it for and I'm pretty sure I was on the correct post that I was replicating when I initiated the research note. My number matches @Fastie _Mini Pearl Jam_ post.

**Comment by jywarren** [comment]
This actually occurs on this line: https://github.com/publiclab/plots2/blob/master/app/views/notes/_responses.html.erb#L26

It's using the `insert_extras` helper, which is older... maybe that's what's broken... https://github.com/publiclab/plots2/blob/master/app/helpers/application_helper.rb#L30

It's also using `response_count` -- https://github.com/publiclab/plots2/blob/f1f9f2cbfa2d9f58e08d246c47ec0f20f1184c9d/app/models/node.rb#L312

**Comment by jywarren** [comment]
OK! Found a deeper bug which may be the cause -- wrote tests for this in https://github.com/publiclab/plots2/pull/1618 and solving there...

**Comment by jywarren** [comment]
I believe we may have a fix for this. Waiting on #1618 and then we'll see if #1567 passes too.

**Comment by jywarren** [comment]
OK #1618 is resolved! Now seeing if #1567 works too.

**Comment by jywarren** [comment]
Fixed!!
